lib/modules/platform/gitea/gitea-helper.spec.ts

```diff
@@ -5,7 +5,8 @@ import { toBase64 } from '../../../util/string';
 import * as ght from './gitea-helper';
 
 describe('modules/platform/gitea/gitea-helper', () => {
-  const baseUrl = 'https://gitea.renovatebot.com/api/v1';
+  const giteaApiHost = 'https://gitea.renovatebot.com/';
+  const basePath = '/api/v1';
```

Comment:
i want only the test base url reverted, so we have less changes. 

yes, please trim in init platform but append API on gitea-helper

Reply:
I've refcatored the gitea-helper and regex replacement as suggested. Nevertheless, I'm not sure I get the point with the gitea-helper test file. In [setBaseUrl()](https://github.com/renovatebot/renovate/blob/3e312fd641dc9129b58453ff383d12705dba8995/lib/modules/platform/gitea/gitea-helper.spec.ts#L144) the previous variable `baseUrl = 'https://gitea.renovatebot.com/api/v1';` was used. If you want to remove the pathname resulting in `baseUrl = 'https://gitea.renovatebot.com';`, it's necessary to adjust all the tests using a httpMock instance (resulting in the same amount of changes). The variable baseUrl must not contain the api path because setBaseUrl + the gitea helper logic would cause the api path to be appended twice. As an alternative, I followed the pattern in [the presets test file](https://github.com/renovatebot/renovate/blob/main/lib/config/presets/gitea/index.spec.ts) using giteaApiHost and basePath variables.

Reply:
leave `baseUrl` for the tests and pass `giteaApiHost` to `setBaseUrl`. this should remove any test changes besides `setBaseUrl` 🤗